app/index.js

```diff
@@ -1,17 +1,34 @@
-import { setup } from './insights.js'
-import 'log-timestamp'
-import { createServer } from './server.js'
+const { setup } = require('./insights.js')
+require('log-timestamp')
+const messaging = require('./messaging/index.js')
+const { createServer } = require('./server.js')
+
+let serverLogger
 
 const init = async () => {
   const server = await createServer()
   await server.start()
-  console.log('Server running on %s', server.info.uri)
+  serverLogger = server.logger
```

Comment:
Just as a general practice. Prefer not to rename things like this as it makes them harder to find in the codebase.

`server.logger.warn` is always `server.logger.warn` but `serverLogger.warn` only exists here. If we do feel we want to take something out of it's context, better to deconstruct? `{ logger } = server`;

Reply:
Will rename to logger